data/crac-creation/crac-creator-fb-constraint/src/main/java/com/powsybl/openrao/data/craccreation/creator/fbconstraint/FbConstraintImporter.java

```diff
@@ -103,11 +108,16 @@ public boolean exists(String s, InputStream inputStream) {
         }
     }
 
+    @Override
+    public CracCreationContext importData(InputStream inputStream, CracCreationParameters cracCreationParameters, Network network, OffsetDateTime offsetDateTime) {
+        return new FbConstraintCracCreator().createCrac(importNativeCrac(inputStream), network, offsetDateTime, cracCreationParameters);
+    }
+
     private int flowBasedDocumentVersion(InputStream inputStream) {
         int schemaVersion = Integer.MIN_VALUE;
 
         try (InputStreamReader isr = new InputStreamReader(inputStream, StandardCharsets.UTF_8);
-            BufferedReader br = new BufferedReader(isr)) {
+             BufferedReader br = new BufferedReader(isr)) {
```

Comment:
Is it a trailing space?

Reply:
no it's a continuation of the parenthesis on the previous line